pos_show_clock/static/src/xml/ChromeWidgets/Clock.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="Clock" owl="1">
+        <div class="oe_status time">
+            <span>
+                <span t-if="!env.isMobile" t-esc="time" class="time" />
+            </span>
+        </div>
+    </t>
+
+    <t
+        t-name="Chrome"
+        t-inherit="point_of_sale.Chrome"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//SyncNotification" position="before">
+            <Clock />
+        </xpath>
+    </t>
```

Comment:
```suggestion
    <t t-name="Clock" owl="1">
        <div class="oe_status time">
            <span>
                <span t-esc="time" class="time" />
            </span>
        </div>
    </t>

    <t
        t-name="Chrome"
        t-inherit="point_of_sale.Chrome"
        t-inherit-mode="extension"
        owl="1"
    >
        <xpath expr="//SyncNotification" position="before">
            <Clock t-if="!env.isMobile" />
        </xpath>
    </t>
```

Better to not render the component itself. That prevents the js code execution as well, as the component will never be mounted if it's on mobile.